packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -142,9 +145,9 @@ const Dropdown = React.forwardRef(function Dropdown(
   return (
     <div className={wrapperClasses} {...other}>
       {titleText && (
-        <label className={titleClasses} {...getLabelProps()}>
+        <Text as="label" className={titleClasses} {...getLabelProps()}>
```

Comment:
Same question here since it can be `PropTypes.node`

Reply:
Yep, will remove